connect/client/fluent.py

```diff
@@ -274,12 +291,14 @@ def __init__(self, *args, **kwargs):
     def session(self):
         value = self._session.get()
         if not value:
-            value = httpx.AsyncClient(
-                transport=_ASYNC_TRANSPORTS.setdefault(
-                    self.endpoint,
-                    httpx.AsyncHTTPTransport(verify=_SSL_CONTEXT),
-                ),
-            )
+            transport = _ASYNC_TRANSPORTS.get(self.endpoint)
```

Comment:
also fixed how transports are reused, because a new transport object was created every time even if the existing one for the endpoint was used.